test/TestActivities.cs

```diff
@@ -86,9 +86,11 @@ public static double BindToDouble([ActivityTrigger] double value)
             await input.CopyToAsync(output);
         }
 
-        public static HttpManagementPayload GetAndPassHttpManagementPayload([ActivityTrigger] DurableActivityContext ctx)
+        public static HttpManagementPayload GetAndPassHttpManagementPayload(

```

Comment:
Let's rename this to `GetAndReturnHttpManagementPayload`.

Reply:
Thank you! I will change it.